pkgs/applications/misc/osm2xmap/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, libroxml, proj, libyamlcpp, boost } :
+stdenv.mkDerivation {
+
+  name = "osm2xmap-2.0";
+
+  src = fetchurl {
+    url = "https://github.com/sembruk/osm2xmap/archive/v2.0.tar.gz";
+    sha256 = "0xpsslh3hzkqz4rs4975fh6yay6m8x5y9fpi42hjv98fvh9m3jwb";
+  };
+
+  preConfigure = ''
+    sed -i 's@/usr/share/osm2xmap/@\$(out)/share/@' Makefile
+    sed -i 's@/usr/bin@/bin@' Makefile
+    sed -i 's@/usr/share/man/man1@/share/man/man1@' Makefile
+    sed -i 's@INSTALL_SHAREDIR = $(DESTDIR)$(SHAREDIR)@INSTALL_SHAREDIR = \$(out)/share/@' Makefile
+    sed -i '/GIT_VERSION =/d' Makefile
+    sed -i '/GIT_TIMESTAMP =/d' Makefile
+    export DESTDIR=$out
+    export GIT_VERSION=2.0
+    '';
+
+  preBuild = ''
+    '';
```

Comment:
I might be wrong but I don't think this overrides it, you might need to pass `:` to do that, see #29583.